ale_linters/typescript/xo.vim

```diff
@@ -11,7 +11,7 @@ endfunction
 function! ale_linters#typescript#xo#GetCommand(buffer) abort
     return ale#Escape(ale_linters#typescript#xo#GetExecutable(a:buffer))
     \   . ale#Pad(ale#Var(a:buffer, 'typescript_xo_options'))
-    \   . ' --reporter unix --stdin --stdin-filename %s'
+    \   . ' --reporter json --stdin --stdin-filename %s'
```

Comment:
The reporter was updated here, but not the handler below.

Reply:
Yup. I must've forgotten it when I added the "old" code back in.

I'll fix this stuff tomorrow.